pkg/cortex/modules.go

```diff
@@ -80,6 +81,13 @@ const (
 	All                      string = "all"
 )
 
+func newDefaultConfig() *Config {
+	defaultConfig := &Config{}
+	defaultFS := flag.NewFlagSet("", flag.PanicOnError)
+	defaultConfig.RegisterFlags(defaultFS)
```

Comment:
[non blocking, just FYI] You can achieve the same with `flagext.DefaultValues()`.